app/views/layouts/notebook.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+  <html lang="<%= I18n.locale %>">
+  <head>
+    <% if @notebook_type == 'ipynb' %>
+      <%= stylesheet_link_tag('jupyter-notebook') %>
+    <% else %>
```

Comment:
I'd prefer using an `elsif` for the rmarkdown type, so that the "else" case is there's no style tag linked at all. Not a big deal, but I like being explicit here.